superset/dashboards/api.py

```diff
@@ -173,7 +177,9 @@ class DashboardRestApi(BaseSupersetModelRestApi):
     dashboard_get_response_schema = DashboardGetResponseSchema()
     dashboard_dataset_schema = DashboardDatasetSchema()
 
-    base_filters = [["slice", DashboardFilter, lambda: []]]
+    base_filters = [
+        ["id", DashboardAccessFilter, lambda: []],
```

Comment:
The first parameter doesn't really matter since `DashboardAccessFilter` doesn't use the column name from `RestApi` class. It's confusing enough, but using `slice` feels more confusing.

Reply:
`"slice"` looks like a bad copy/pasta. 👍